<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LiftPage</class>
 <widget class="QWidget" name="LiftPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="BTN_FINDHOME">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Find Home</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_CANCHECK">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>CAN Check</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_FETOFF">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>FET Off</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_RIFT_UP">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Up</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_STATUS">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_REF">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_CTRLON">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Ctrl On</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_REFOFF">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Ref Off</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_CTRLOFF">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Ctrl Off</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_FETON">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>FET On</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_REFON">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Ref On</string>
   </property>
  </widget>
  <widget class="QLabel" name="LABEL_ENCODER">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>30</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Enc</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_STOP">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>130</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BTN_DOWN">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>60</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Down</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_REF_STATUS">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LABEL_REF">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>30</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ref</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_ENCODER">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_GOAL">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>1000</string>
   </property>
  </widget>
  <widget class="QLabel" name="LABEL_GOAL">
   <property name="geometry">
    <rect>
     <x>273</x>
     <y>111</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Goal</string>
   </property>
  </widget>
  <widget class="QLabel" name="mm">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>110</y>
     <width>67</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>mm</string>
   </property>
  </widget>
  <widget class="QLabel" name="LABEL_PULSE">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>67</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
